src/RelayServer.ts

```diff
@@ -380,64 +379,34 @@ export class RelayServer extends EventEmitter {
     envelopingTransaction: EnvelopingTxRequest
   ): Promise<BigNumber> {
     log.debug(
-      `Enveloping transaction: ${JSON.stringify(
+      `***Enveloping transaction: ${JSON.stringify(
```

Comment:
I'd remove this change. If we need a specific format for some logs, I'd create one function that format them, instead of doing it on a single log basis.
E.g.:
```
const formatLog = (msg: string) => `***${msg}`;
```

Reply:
Done.